inginious/frontend/plugins/contact_page/static/js/contact_page_form.js

```diff
@@ -116,69 +162,78 @@ class ContactPageForm {
     }
 
     sendRequest() {
+        const self = this;
         jQuery.ajax({
             method: "POST",
             data: {
                 "subject_id": this.getOptionSelected(),
                 "email": this.emailInput.val(),
                 "name": this.nameInput.val(),
                 "courseName": this.courseNameInput.val(),
+                "courseGroup":this.courseGroupInput.val(),
                 "textarea": this.textarea.val()
             },
-            success: function (data) {
-                new MessageBox(ALERT_SPACE_ID, "The message has been sent", "info", false);
+            success: function (data) { 
+                self.alertOnModal.addClass('alert-success');
+                self.textOnAlertSuccess.show();
+                self.addCloseListenerToOnCloseModal();
+                self.modalSendMessage.modal("show"); 
             },
             error: function (request, status, error) {
-                new MessageBox(ALERT_SPACE_ID, "The message could not be sent", "danger", false);
+                self.alertOnModal.addClass('alert-danger');
+                self.textOnAlertFail.show();
+                self.addCloseListenerToOnCloseModal();
+                self.modalSendMessage.modal("show");
+                self.errorText.text(request.status+ " " + error);
+                self.details.show();
             }
         });
     }
 
     addErrorStyle(inputObj, errorText) {
```

Comment:
```suggestion
    addErrorStyle(inputObj, errorMessageId) {
```
Maybe this variable name can be changed since ID is used instead of message text.